aws_extender_cli.py

```diff
@@ -39,8 +39,8 @@ def init_clients(keys_path, service):
         with open(keys_path, 'r') as fkeys:
             keys = fkeys.read()
             try:
-                access_key = re.search(r'AccessKeyId[=:]([^\n]+)', keys, re.I).group(1)
-                secret_key = re.search(r'SecretKey[=:]([^\n]+)', keys, re.I).group(1)
+                access_key = re.search(r'AccessKeyId[=:]([^\n\r]+)', keys, re.I).group(1)
+                secret_key = re.search(r'SecretKey[=:]([^\n\r]+)', keys, re.I).group(1)
```

Comment:
Hi, @ciara1234!

First, thank you for the pull request. I see that the end-of-line convention on Windows systems is `\r\n` (in contrast to `\n` on Unix systems)....

I guess we could be even more restrictive and use `r'AccessKeyId[=:]([^\s]+)'` instead (so all whitespace characters, especially trailing spaces, are excluded).

In any case, the changes here should definitely be merged.

Reply:
Awesome, suppressing all whitespace sounds very neat and clean. Thanks :-)